scene_graph/vrd_tutorial.ipynb

```diff
@@ -565,7 +562,96 @@
    "cell_type": "code",
    "execution_count": 12,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
nit: can the output be cleaned before merging with master

Reply:
Whoops sorry didn't see this. Hasn't our MO been to have the outputs rendered?